lib/rubocop/cop/rspec/mixin/location_help.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Helper methods to location.
+      module LocationHelp
+        # @param node [RuboCop::AST::SendNode]
+        # @return [Parser::Source::Range]
+        # @example
+        #   foo 1, 2
+        #      ^^^^^
+        def self.arguments_with_whitespace(node)
```

Comment:
I think we/RuboCop normally use `module_function` on modules that aren’t meant to be included/extended anywhere:
```suggestion
        module_function

        # @param node [RuboCop::AST::SendNode]
        # @return [Parser::Source::Range]
        # @example
        #   foo 1, 2
        #      ^^^^^
        def arguments_with_whitespace(node)
```

Reply:
I see, if that is the way it is with us/RuboCop, it seemed like a good to follow it.
I updated this PR.